db/migrate/1000016_aggregated_migrations.rb

```diff
@@ -399,6 +407,33 @@ class AggregatedMigrations < SquashedMigration
     20241001205821_add_standard_global_role
     20241002151949_remove_hide_mail_from_user_preferences
     20241015081341_remove_obsolete_design_variables
```

Comment:
Those references to the migrations squashed before can be removed as they have been dealt with in OP 16.